libs/shared/ui/src/lib/components/tag/tag.tsx

```diff
@@ -13,6 +13,9 @@ export interface TagProps {
   fontWeight?: string
 }
 
+/**
+ * @deprecated Should be replaced by Badge component
+ */
 export function Tag({
```

Comment:
I think you can remove it, we don't use it right now

Reply:
This is still used for `TagCommit` which should be rewritten from scratch.
I didn't want to do too much refactor so I leave this one in place for this case